code/game/objects/structures/salvaging.dm

```diff
@@ -32,6 +34,17 @@
 		qdel(src)
 	return TRUE
 
+/obj/structure/salvageable/default_unfasten_wrench(mob/living/user, obj/item/I, time = 20)
+	. = ..()
+	var/obj/frame = frame_type
+	if(. == SUCCESSFUL_UNFASTEN)
+		frame.anchored = anchored
```

Comment:
I don't think this'll work, since the frame type's just a type, not an actual thing. Handling the anchoring should be fine with what you have in dismantle() already